### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by IvanGuculyak**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    arguments = [word for word in command.split()]
```
Comment (Vasyl-Smutok) [inline app/main.py:2]:
```suggestion
    cp_command, current_file, future_file = command.split()
```

**Review by Vasyl-Smutok — CHANGES_REQUESTED** [review]
image

File: app/main.py [hunk 2]
```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp_command, current_file, future_file = command.split()
```
Comment (ArsenPidhoretskyi) [inline app/main.py:2]: Your code will broke in next cases:
command = 
"cp"
"cp file1.txt"
"cp file1.txt file2.txt file3.txt"

File: app/main.py [hunk 3]
```diff
@@ -1 +1,6 @@
...
+    if current_file == future_file and cp_command == "cp":
```
Comment (ArsenPidhoretskyi) [inline app/main.py:3]: if current_file == future_file but command will not equal to cp -> we will do copy
Strange logic

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by katevasilenko — APPROVED** [review]
Good job!

**Pull request closed without merging**